test/common/testlib.py

```diff
@@ -784,9 +788,6 @@ def skipImage(reason, *args):
     return lambda func: func
 
 class Policy(object):
-    def __init__(self):
-        self.retryable = True
```

Comment:
Why remove this? There can be tests that shouldn't be retried and we will run into them. For those, we'll have to adapt the policy, and for such cases it'll be good to have the variable in place.